packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -281,7 +291,14 @@ void main() {
         )).captured;
         expect(captures[0], 'run');
         final Map<String, String> parameters = captures[1] as Map<String, String>;
-        expect(parameters['cd4'], 'ios');
+
+        expect(parameters['cd3'], 'false'); // commandRunIsEmulator
+        expect(parameters['cd4'], 'ios'); // commandRunTargetName
+        expect(parameters['cd15'], 'swift'); // commandRunProjectHostLanguage
+        expect(parameters['cd22'], 'iOS 13'); // commandRunTargetOsVersion
+        expect(parameters['cd23'], 'debug'); // commandRunModeName
+        expect(parameters['cd18'], 'false'); // commandRunProjectModule
+        expect(parameters.containsKey('cd45'), false); // commandRunAndroidEmbeddingVersion
       }, overrides: <Type, Generator>{
```

Comment:
Do we already test the other scenarios?

Reply:
I don't see these particular keys tested anywhere else, unless I'm missing something.  I see `CustomDimensions.hot*` `CustomDimensions.enabledFlutterFeatures`, `CustomDimensions.buildEvent*` and cd27-43.  I'll switch this to `cdKey()`.